mreg_cli/utilities/api.py

```diff
@@ -333,6 +335,7 @@ def get(path: str, params: QueryParams | None = ..., *, ok404: bool) -> Response
 def get(path: str, params: QueryParams | None = ...) -> Response: ...
 
 
+@cache.memoize(expire=300, tag="api")
```

Comment:
Should we possibly only cache calls where we get 2XX from the server?

Reply:
Don't we raise an exception for non-2xx responses? In that case the memoized function should never get to the point at which it actually writes to the cache.

https://github.com/grantjenks/python-diskcache/blob/ebfa37cd99d7ef716ec452ad8af4b4276a8e2233/diskcache/core.py#L1874-L1877

The library just calls the decorated function with the given arguments, so if an exception is raised, it's propagated up the stack before the results are cached.